checkout/frames-android
# PIMOB: 2109: Allow injection of cardholder name field in paymentformConfig

## Issue

[PIMOB-2109](https://checkout.atlassian.net/browse/PIMOB-2109)

## Proposed changes
1 Add cardHolderName in PaymentFormConfig
2 Load the cardholdername field in the Payment form from PaymentFormConfig
3. Refactored existing unit test for paymentformstatemanager field

## Test Steps
1. add cardHolderName in PaymentFormConfig
2. Verify the paymentform details' cardholder name is updated
[Screen_recording_20230816_114904.webm](https://github.com/checkout/frames-android/assets/114917119/d2d5d82f-6a42-4608-af64-e8736a602061)



## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._


## Further comments

_If this is a relatively large or complex change, kick off the discussion by explaining why you choose the solution you did and what alternatives you considered, etc..._


[PIMOB-2109]: https://checkout.atlassian.net/browse/PIMOB-2109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
